### PR TITLE
Chore/160451490/hound ci

### PR DESCRIPTION
**Pull request by udochukwu**
#### What does this PR do?
This PR is to integrate Hound CI to the Github repository
#### Description of Task to be completed?

- Login to the HoundCI website with your Github username and activate the repo you intend to integrate

- Create a **`.hound.yml`** file and specify the rules & configurations

- Add a Badge to the ReadMe file

#### How should this be manually tested?

- N/A

#### What are the relevant pivotal tracker stories?
Story type: 

- chore

- Story title: Integrate HoundCI with a badge attached to the readme file.

- Story ID: **160451490**

#### Screenshots (if appropriate)

- N/A

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (s-akinrele) [inline .hound.yml]: add a new line @udochukwu

Comment (udochukwu) [reply .hound.yml]: Hii @s-akinrele, please where am I meant to add a new line.

**Comment by kampkelly** [comment]
Nice work. It looks good to me!

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
```
Comment (onwuvic) [inline .hound.yml]: Please add a newline at the end of this file.

**Review by onwuvic — COMMENTED** [review]
LTGM

**Pull request merged**